README_YORICK.md

```diff
@@ -0,0 +1,4 @@
+# Yorick-specific Notes
```

Comment:
This file had spoken about the (now removed) `yk_hwt` compile-time gate. Now it talks about the environment var to enable the MIR ELF section.